src/breadcrumb.component.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import { Button } from './button.component'
+import { getColor } from './provider.component'
+
+import useWindowSize from '@rehooks/window-size'
+
+import { Anchor } from 'grommet'
+
+
+export function Breadcrumb( props ) {
+  
+  const windowSize = useWindowSize()
+  const showFull = ( windowSize.innerWidth > 900 ) ? true : false;
+
+  return (
+      <div style={{minHeight:'2rem'}}>
+
+        { (!showFull) ?
+          <StyledButton
+          label={"Übersicht Prozentrechnung"}
+          className={props.className}
+          a11yTitle={props.title}
+          plain
+          iconName={'arrow-circle-left'}
+          
+          backgroundColor={getColor('lighterblue')}
+          activeBackgroundColor={getColor('brand')}
+        />
+          :
+          <BreadcrumbList>
+            <Anchor href="#">Mathematik </Anchor>
+            &nbsp;>&nbsp;<Anchor href="#">Terme und Gleichungen</Anchor>
+            &nbsp;>&nbsp;<Anchor href="#">Terme und Variablen</Anchor>
+            &nbsp;>&nbsp;<Anchor href="#">Zusammenfassen, Ausmultiplizieren, Faktorisieren</Anchor>
+          </BreadcrumbList>
+        }
+    </div>
+  )
+}
+
+const BreadcrumbList = styled.div `
+  position: absolute;
+  left: 2rem;
+  margin-top: -1rem;
+  font-size: 1rem;
+  color: #ddd;
+
+  > a {
```

Comment:
I'd prefer something like `BreadcrumbAnchor = styled(Anchor)` instead. Then you don't have to make the connection between `a` and `Anchor`, don't have to know that an `Anchor` renders an `a` somewhere and also don't have to consider weird css inheritance (what if `Anchor` already specifies a color?)